README.md

```diff
@@ -1,31 +1,31 @@
 # CanCat
-swiss army knife of Controller Area Networks (CAN) often used in cars and building automation, etc...
+CanCat is the swiss army knife of Controller Area Networks (CAN) often used in cars and building automation, etc.
 
-made possible through collaboration with my friends at GRIMM (SMFS), most notably Matt Carpenter and Tim Brom.
+CanCat is made possible through collaboration with my friends at GRIMM (SMFS), most notably Matt Carpenter and Tim Brom.
 
 CanCat is made up of two parts:
 1) Firmware for a compatible CAN-transceiver
 2) Python client to talk to the tool
 
-it is a lot like the architecture for RfCat, and we may make it moreso.  currently we're sticking with the original design by Matt and team.
+CanCat is a lot like the architecture for RfCat, and we may make it moreso. Currently we're sticking with the original design by Matt and team.
 
-currently supported CAN-transceiver combinations:
+The currently supported CAN-transceiver combinations:
 * Arduino with SeeedStudio's CANBUS Shield
 * Arduino DUE with Togglebit.net's CAN shield
```

Comment:
sorry to glom onto your PR, but would you mind adding in the Macchina M2 (with a link to the macchina.cc page for both under-hood and under-dash versions)?

Reply:
Ok.